src/lib.rs

```diff
@@ -1619,6 +1619,30 @@ impl<B: BitBlock> BitVec<B> {
     pub fn shrink_to_fit(&mut self) {
         self.storage.shrink_to_fit();
     }
+
+    /// Inserts a given bit at index `at`
+    ///
+    /// # Examples
+    ///```
+    /// use bit_vec::BitVec;
+    /// let mut b = BitVec::new();
```

Comment:
Could have a newline after the import, because imports are kind of not program code and no point adding space between program code parts if you have no space from imports